packages/provider.component/src/ComponentProvider.ts

```diff
@@ -26,26 +26,31 @@ export default class ComponentProvider extends Provider {
 
   /**
    * Convert <slot name='X'> to <!-- ko slot: 'X' --><!-- /ko -->
-   * @param {HTMLElement} node
+   * @param {Element} node
    */
-  preprocessNode (node) {
+  preprocessNode (node : Element) {
     if (node.tagName === 'SLOT') {
       const parent = node.parentNode
       const slotName = node.getAttribute('name') || ''
       const openNode = document.createComment(`ko slot: "${slotName}"`)
       const closeNode = document.createComment('/ko')
+
+      if(!parent)
```

Comment:
should use `if (...) { ... }` by convention / clarity / lint

Reply:
(not a blocker, but noting)

Reply:
Done